docs/scos/dev/guidelines/project-development-guidelines.md

```diff
@@ -20,6 +20,16 @@ Spryker OS exposes codebase Projects, which enables a high level of customizatio
 
 The project development, the team is free to decide what approach to use. Spryker recommends considering **Configuration**, **Plug and Play**, and **Project modules** first to get maximum from the Spryker OS codebase, atomic releases, leverage minimum efforts for the integration of the new features and keeping system up to date.
 
+Before jumping on the strategies definition 
```

Comment:
Before jumping on the strategies definition, check out the table of Spryker OS update support to make the best decision about the customization strategy